src/app/core/views/portinfo/analog/index.ts

```diff
@@ -41,4 +41,16 @@ export const AnalogPortInfo: PortInfoRecord<AnalogComponent> = {
             },
         },
     },
+    "Capacitor": {
+        Default:       DefaultAnalogPort,
+        InitialConfig: "2",
+        AllowChanges:  false,
+
+        Positions: {
```

Comment:
Is this compiling? I'm pretty sure this is supposed to be `PositionConfigs`, make sure you are up-to-date with `model_refactor`

Reply:
up to date now